src/release-downloader.ts

```diff
@@ -26,11 +26,20 @@ export class ReleaseDownloader {
 
     if (downloadSettings.isLatest) {
       ghRelease = await this.getlatestRelease(downloadSettings.sourceRepoPath)
-    } else {
+    } else if (downloadSettings.tag !== "") {
       ghRelease = await this.getReleaseByTag(
         downloadSettings.sourceRepoPath,
         downloadSettings.tag
       )
+    } else if (downloadSettings.id !== "") {
+      ghRelease = await this.getReleaseById(
+        downloadSettings.sourceRepoPath,
+        downloadSettings.id
+      )
+    } else {
+      throw new Error(
+        "Config error: Please input a valid tag or release ID, or specify `latest`"
```

Comment:
It would be nice if you could add a test for this case as well. A similar one can be found at https://github.com/robinraju/release-downloader/blob/main/__tests__/main.test.ts#L115-L129

Reply:
Also, you need to update `input-helper.ts` to read the new input from the `action.yml`
https://github.com/kevcube/release-downloader/blob/main/src/input-helper.ts#L25-L33

```ts
  const releaseId = core.getInput("releaseId")
  
  if (latestFlag && ghTag.length > 0 && release.length > 0) {
    throw new Error(
      `Invalid inputs. latest=${latestFlag} and tag=${ghTag} releaseId=${releaseId} can't coexist`
    )
  }

  downloadSettings.id = releaseId
```